browser-test/src/file.test.ts

```diff
@@ -0,0 +1,131 @@
+import {
+  AdminPrograms,
+  AdminQuestions,
+  ApplicantQuestions,
+  loginAsAdmin,
+  loginAsGuest,
+  logout,
+  resetSession,
+  selectApplicantLanguage,
+  startSession,
+} from './support'
+
+describe('file upload applicant flow', () => {
```

Comment:
I don't see a case for 2 upload questions.  Was that not actually doable after all?

The name question tests should have examples for you if that's helpful.

Reply:
The UI actually appears to disallow adding more than one file upload question to a single screen. I originally wrote a test which attempted this and ran into that failure.

Reply:
Ah sorry, for some reason I thought Name was similar in only allowing one.

Reply:
No worries at all. Resolving this.